src/router/routes.tsx

```
@@ -1,7 +1,8 @@
 import { createBrowserRouter } from 'react-router-dom';
 
 import { AppLayout, ContentLayout } from '@layouts/index';
-import { Dashboard } from '@pages/index';
+import { Error } from '@pages/Error';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)